File: common_library/src/main/java/com/phone/common_library/manager/ResourcesManager.java
package com.phone.common_library.manager;

import androidx.core.content.ContextCompat;

import com.phone.common_library.BaseApplication;

public class ResourcesManager {

    public static String getString(int stringId) {
        return BaseApplication.getInstance().getResources().getString(stringId);
    }

    public static int getColor(int colorId) {
//        return BaseApplication.getInstance().getResources().getColor(colorId);
        return ContextCompat.getColor(BaseApplication.getInstance(), colorId);
    }
}

File: library_common/src/main/java/com/phone/common_library/manager/ResourcesManager.kt
package com.phone.common_library.manager

import androidx.core.content.ContextCompat
import com.phone.common_library.BaseApplication

object ResourcesManager {

    fun getString(stringId: Int): String {
        return BaseApplication.get().resources.getString(stringId)
    }

    fun getColor(colorId: Int): Int {
//        return BaseApplication.get().getResources().getColor(colorId);
        return ContextCompat.getColor(BaseApplication.get(), colorId)
    }
}